starlite/middleware/session/redis_backend.py

```diff
@@ -0,0 +1,55 @@
+from typing import Type, Union
+
+from redis.asyncio import Redis
+
+from starlite.middleware.session.base import ServerSideBackend, ServerSideSessionConfig
+
+
+class RedisBackend(ServerSideBackend["RedisBackendConfig"]):
+    def __init__(self, config: "RedisBackendConfig") -> None:
+        """Session backend to store data in redis.
+
+        Notes:
+            - Requires `redis`. Install with `pip install starlite[redis]`
+        """
+        super().__init__(config=config)
+        self.redis = config.redis
```

Comment:
Note - the class is missing `__scopes__ = ()` - please make sure all classes that can implement this, do implement it.

Reply:
I assume you mean `__slots__`?